website/_mechanics/basics/02-terminology.md

```diff
@@ -176,3 +176,21 @@ See [Slopesurfing](#slopesurfing)
 A tech that enters the wrong state when jumping, allowing an earlier wallrun.
 
 See [Raiserunning](#raiserunning)
+
+{:.no_toc}
+### Clip
+Certain configurations of geometry allow the player to pass through solid walls.
+
+{:.no_toc}
+### Magnet
+Jumping mid-air will "magnet" you to walls approximately within half of the width (TODO: fact check?) of your characters hitbox.
+
+This is often negatively experienced for the first time for many players in [Backup Shift](#maps-level-backup-shift) (TODO: earlier levels?)
+
```

Comment:
Possibly mention how Wall Magnets can be used to positive effect. for example, Snap Jumps / Snap Dashes (alternate names for intentional wall-magneting) can make Giga Walls easier

Reply:
```suggestion
However, it can also be used for positive effect as is often seen in the Walls (last) section of [Giga Difficult](#maps-level-giga-difficult).
```